README.md

```diff
@@ -19,7 +19,7 @@ Think about contributing? That's great, but please do read our [contributing gui
 * [Blog post: building rednoseday.com on Drupal 8](https://technology.comicrelief.com/2016/12/21/building-rednoseday-com-on-drupal-8/)
 * [Video DrupalCon Baltimore 2017: building reusable websites on Drupal 8](https://technology.comicrelief.com/2017/05/12/drupalcon-baltimore-talk-slides/)
 * [Blog post: Why we made our platform product open-source](https://technology.comicrelief.com/2017/07/07/why-we-made-our-platform-product-open-source/)
-* [Release process](docs/release.md)
+* [Release process](docs/releases.md)
```

Comment:
Add in platform.sh docs around preview branches

Reply:
Add in a link to the testing docs and make sure these docs are up to date

Reply:
Why do we need a testing docs? Nowadays run the tests are easy and if you follow the official documentation you won't have any issue (and before our tests require some conditions, and they were in strange location, etc...)


Maybe we should mention that we use behat for our tests and a link to the documentation?

Reply:
@Saphyel where can I find how to run the tests? Don't think we've got any documentation for that

Reply:
We use behat... any question about behat or how to use it should be in behat not in our repo...